creator/extract_configs/templates/participant_config.py

```diff
@@ -1,6 +1,23 @@
 """
 This is an extract configuration for the `Participant Demographic and
 Administrative File` template
+
+See: https://www.notion.so/d3b/Internal-File-Types-and-Guidelines-6edd92087f04445e84a236202831edbd#aa2c7d8a50164a08ab685b42b784293e
+
+Required Columns:
+    Participant ID
+    Consent Short Name
+    Affected Status
+    Administrative Gender
+    Ethnicity
+    Age at Study Enrollment Value
+    Age at Study Enrollment Units
+    Deceased
```

Comment:
Seems like Deceased isn't used in the ingest? Does that make it Optional?

Reply:
Ah it should be used, I probably just forgot. The required vs optional columns are defined in the file type definitions here: https://www.notion.so/d3b/File-Types-and-Guidelines-fc5bd4390fa54de5a70b550d73779de9